include/dlaf/eigensolver/reduction_to_band/impl.h

```diff
@@ -939,10 +931,9 @@ struct ComputePanelHelper<Backend::GPU, Device::GPU, T> {
 }
 
 /// Local implementation of reduction to band
-/// @return a vector of shared futures of vectors, where each inner vector contains a block of taus
+/// @return a column vector of taus with block size mat_a.blockSize().cols()
```

Comment:
Doxygen documentation should not be here.

At most I would keep the comment
`// Local implementation of reduction to band`

Reply:
https://github.com/eth-cscs/DLA-Future/pull/908/commits/4322a278222a47da74af0cd9737df40c32bde8b4